atom/browser/api/lib/auto-updater/auto-updater-win.js

```diff
@@ -2,70 +2,62 @@ const app = require('electron').app;
 const EventEmitter = require('events').EventEmitter;
 const url = require('url');
 const squirrelUpdate = require('./squirrel-update-win');
+const util = require('util');
 
-var extend = function(child, parent) { for (var key in parent) { if (hasProp.call(parent, key)) child[key] = parent[key]; } function ctor() { this.constructor = child; } ctor.prototype = parent.prototype; child.prototype = new ctor(); child.__super__ = parent.prototype; return child; },
-hasProp = {}.hasOwnProperty;
+function AutoUpdater() {
+  AutoUpdater.super_.call(this);
```

Comment:
So if `EventEmitter.call(this)` is called here then if the parent class changes, two places would need to be updated, this line and the `util.inherits` call.

Using `super_` seemed clear to me and would make future changes easier to make, less error prone, and easier to review.